Source/control.cpp

```diff
@@ -817,6 +758,58 @@ void DrawSpellList(const Surface &out)
 	}
 }
 
+std::vector<SpellListItem> GetSpellListItems()
+{
+	std::vector<SpellListItem> spellListItems;

```

Comment:
I know you've simply moved an existing function but it'd be good to clean this up to avoid allocating a vector on every frame (in a separate PR).